packages/cli/lib/zeroYaml/utils.ts

```diff
@@ -10,7 +10,7 @@ export async function syncTsConfig({ fullPath }: { fullPath: string }) {
     await fs.promises.writeFile(path.join(fullPath, 'tsconfig.json'), await fs.promises.readFile(path.join(exampleFolder, 'tsconfig.json')));
 }
 
-export function fileErrorToText({ filePath, msg, line, character }: { filePath: string; msg: string; line?: number; character?: number }) {
+export function fileErrorToText({ filePath, msg, line, character }: { filePath: string; msg: string; line?: number | undefined; character?: number }) {
     return `${chalk.red('err')} - ${chalk.blue(filePath)}${line ? chalk.yellow(`:${line + 1}${character ? `:${character + 1}` : ''}`) : ''} \r\n  ${msg}\r\n`;
 }
```

Comment:
was already the case but does it mean the console can show `undefined` for line?